src/util/is-web-assembly-supported.ts

```diff
@@ -0,0 +1,16 @@
+const isWebAssemblySupported = () => {
+  try {
+    if (typeof WebAssembly === 'object' && typeof WebAssembly.instantiate === 'function') {
+      const module = new WebAssembly.Module(Uint8Array.of(0x0, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00))
+
+      if (module instanceof WebAssembly.Module) {
+        return new WebAssembly.Instance(module) instanceof WebAssembly.Instance
+      }
+    }
```

Comment:
Where's this sourced from? Wouldn't a simple check for `window.WebAssembly`/`typeof WebAssembly` work here?

Reply:
https://stackoverflow.com/a/47880734 😉

I'm happy to use the simpler solution shown on the last comment on that solution!

Reply:
That answer does a nice job explaining it. Thanks, LGTM